_posts/2020-07-14-moveit-on-windows.md

```diff
@@ -0,0 +1,32 @@
+---
+author: Dave Coleman, PhD
```

Comment:
Typically we don't use titles for blog post authors, but not a big deal, I guess.